std_msgs/Header header
geometry_msgs/Pose pose
int32 detect_seq
float64[] detect_rotation
float64[] detect_intensity
float64 detect_rotation_max
float64 detect_intensity_max
